### PR TITLE
Catgrill

### PR DESCRIPTION
**Pull request by AsciiSquid**
## About The Pull Request

Adds catgrill, a degenerate alternative to standard grilling protocol. Now available for 12000 credits in cargo as contraband.
catgrill
Anything grilled on it is given an "owo" text filter.

## Why It's Good For The Game

Every day we stray further from god. This will likely expedite that process. 

## Changelog
:cl:
add: Added "nic catgrill gf" available in cargo.
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
speedmerge
SPEEDMERGE

**Comment by TheNeoGamer42** [comment]
speedmerge now, its essential for the continued life of boomerstation

**Comment by Victor239** [comment]
That forced speech when you eat grilled foods needs to owo-ified also

File: waspstation/code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -157,3 +157,13 @@
 					/obj/item/am_shielding_container) //10 shields: 3x3 containment and a core
 	crate_name = "antimatter shielding crate"
 	dangerous = TRUE
+/datum/supply_pack/organic/catgrill
+	name = "Felinid-Style Grilling Kit"
+	desc = "Command recieved a request for \"Nic catgrill gf plz\", we're pretty sure this is what that meant."
+	cost = 12000
+	contraband = TRUE
+	crate_type = /obj/structure/closet/crate/trashcart
+	contains = list(/obj/item/stack/sheet/mineral/coal/five,
```
Comment (Superyodeler) [inline waspstation/code/modules/cargo/packs.dm:166]: So you can start burning before you get to hell

**Comment by SynnGraffkin** [comment]
> _Iiiiiiiiiit's_ the end of the world as we know it, and I _feeliiiiiiine_

Fixed it for ya

**Comment by MarkSuckerberg** [comment]
🔫

File: tgstation.dme [hunk 2]
```diff
@@ -3169,6 +3169,7 @@
 #include "waspstation\code\modules\clothing\under\jobs\centcom.dm"
 #include "waspstation\code\modules\clothing\under\jobs\command.dm"
 #include "waspstation\code\modules\clothing\under\jobs\security.dm"
+#include "waspstation\code\modules\food_and_drinks\kitchen_machinery\catgrill.dm"
```
Comment (Superyodeler) [inline tgstation.dme:3172]:
```suggestion
```

File: waspstation/code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -157,3 +157,13 @@
...
+					/obj/machinery/grill/cat,
+					/obj/item/reagent_containers/food/snacks/meat/slab/mouse)
+	crate_name = "catgrill crate"
```
Comment (Superyodeler) [inline waspstation/code/modules/cargo/packs.dm:169]:
```suggestion
```

File: waspstation/code/modules/food_and_drinks/kitchen_machinery/catgrill.dm [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+	var/static/regex/oWo = new("R|L", "g")
+	var/static/regex/Nya = new("N(a|e|i|o|u)", "g")
+	// Forgive me marg for I have sinned
+	grilled_item.name = owo.Replace(grilled_item.name, "w")
+	grilled_item.name = oWo.Replace(grilled_item.name, "w")
+	grilled_item.name = Nya.Replace(grilled_item.name, "Ny$1")
+
+/obj/machinery/grill/cat/finish_grill()
+	..()
+	owoify()
```
Comment (Superyodeler) [inline waspstation/code/modules/food_and_drinks/kitchen_machinery/catgrill.dm:29]:
```suggestion
```

**Comment by MarkSuckerberg** [comment]
we broke him

**Comment by Superyodeler** [comment]
he;[, acsii griffn me

**Comment by MarkSuckerberg** [comment]
he;[

**Pull request merged**